packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,76 @@
+
+:local {
+  .tag {
+    border-radius: 0.2142rem;
+    border-style: var(--terra-tag-border-style, solid);
+    border-width: 0.07142rem;
+    color: var(--terra-tag-color, #434a4d);
+    display: inline-flex;
+    font-size: var(--terra-tag-font-size, 1rem);
+    font-weight: var(--terra-tag-font-weight, normal);
+    line-height: var(--terra-tag-line-height, 1.14285rem);
+    margin-bottom: var(--terra-tag-margin-bottom, 0.357rem);
+    margin-left: var(--terra-tag-margin-left);
+    margin-right: var(--terra-tag-margin-right, 0.714rem);
+    margin-top: var(--terra-tag-margin-top, 0.357rem);
+    padding-bottom: var(--terra-tag-padding-bottom, 0.1428rem);
+    padding-left: var(--terra-tag-padding-left, 0.5rem);
+    padding-right: var(--terra-tag-padding-right, 0.5rem);
+    padding-top: var(--terra-tag-padding-top, 0.1428rem);
+    position: relative;
+    text-align: left;
+    text-decoration: var(--terra-tag-text-decoration, none);
+    text-transform: none;
+    touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
+    user-select: none; // Prevent text selection on tags on mobile devices
```

Comment:
This will also prevent text selection on desktop.

Reply:
Updated the comment to include desktop [here](https://github.com/cerner/terra-core/pull/1306/commits/0c5264f62462ae9bc4cbedbae83c5cac42c28580) .